server/neptune/terraform/command.go

```diff
@@ -0,0 +1,91 @@
+package terraform
+
+import "strings"
+
+type Operation int
+
+const (
+	Init Operation = iota
+	Plan
+	Apply
+)
+
+func (t Operation) String() string {
+	switch t {
+	case Init:
+		return "init"
+	case Plan:
+		return "plan"
+	case Apply:
+		return "apply"
+	}
+	return ""
+}
+
+type CommandArguments struct {
```

Comment:
do these all need to be exported?

Reply:
We can export CommandArguments and keep everything else internal since it's used in the terraform activity.